config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/EntityBuilderHelper.java

```diff
@@ -61,4 +62,8 @@ static Entity getEntityWithMappings(final String type, final String path, final
         }
         return entity;
     }
+
+    static Entity getEntityWithDeleteMappings(final Entity entity){
```

Comment:
Is this mapping resolved by NAME or ID or BOTH?

Reply:
its resolved by name